code/network/multi.h

```diff
@@ -366,6 +366,21 @@ class player;
 
 #define MAX_SHIPS_PER_PACKET		64			// Number of ships in a STATS_MISSION_KILLS or STATS_ALLTIME_KILLS packet
 
+// campaign and mission description packet post-header type info
+#define DESCRIPT_MESSAGE_REQUEST	(1 << 0)
+#define DESCRIPT_PACKET_CAMPAIGN_MESSAGE_1	(1 << 1)
+#define DESCRIPT_PACKET_CAMPAIGN_MESSAGE_2	(1 << 2)
+#define DESCRIPT_PACKET_MISSION_MESSAGE_1	(1 << 3)
+#define DESCRIPT_PACKET_MISSION_MESSAGE_2	(1 << 4)
+#define DESCRIPT_PACKET_NO_ADDENDUM			(1 << 5)
+#define DESCRIPT_PACKET_FINAL_MESSAGE		(1 << 6)
+
+#define MULTI_MAX_DESC_LEN			2*(MAX_PACKET_SIZE - 10)
+
+const char PRE_CAMPAIGN_DESC[23] =	"Campaign Description:\n";
+const char PRE_MISSION_DESC[28]  =	"First Mission:\n";
+const char DOUBLE_NEW_LINE[3]    =  "\n\n";
```

Comment:
These should just be pointers and also not be defined in `multi.h`. Maybe set them in `multimsgs.cpp` and extern when you need in `multiui.cpp`